### PR TITLE
Feat/landing and 404 pages

### PR DESCRIPTION
**Pull request by tonymike99**
In this PR, I have created the landing and 404 pages.

### CODE REVIEW

File: src/components/Aside/Aside.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  grid-area: aside;
+  display: flex;
+  justify-content: unset;
+  align-items: unset;
+  flex-direction: column;
+  padding: 2rem;
+}
+
+a.active {
+  color: forestgreen;
```
Comment (amartya-rs) [inline src/components/Aside/Aside.css:11]: can use `var()`

**Review by amartya-rs — APPROVED** [review]
Hey Tony,
Your code looks fine to me. The .css files are well commented.
Good to merge.

**Review by archit7448 — APPROVED** [review]
Awesome and clean UI 
clean code , work on CSS quit good 
Looks good to merge

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+.footer-code-icon {
+  color: #15aabf;
+}
+
+.footer-github-icon {
+  color: var(--light-color);
+}
+
+.footer-twitter-icon {
+  color: #5aa4d6;
```
Comment (dev-prabhat) [inline src/components/Footer/Footer.css:38]: define color in root as a variable

File: src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            <i className="fa-solid fa-code footer-code-icon" />
+          </span>
+          by
+        </small>
+      </div>
+      <div className="footer-item">
+        <span className="my-name">Tony Mike Auxen</span>
+      </div>
+      <div className="footer-item footer-icons">
+        <a
```
Comment (dev-prabhat) [inline src/components/Footer/Footer.jsx:23]: I think you can avoid anchor , used Link component from react-router-dom

Comment (dev-prabhat) [inline src/components/Footer/Footer.jsx:23]: I think you can avoid anchor , used Link component from react-router-dom

File: src/hooks/context/theme-context.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+const ThemeContext = createContext(defaultObj);
+
+const ThemeProvider = ({ children }) => {
+  const [theme, setTheme] = useState(
+    JSON.parse(localStorage.getItem("storedTheme")) ?? "light-theme"
+  );
+
+  useEffect(() => {
+    localStorage.setItem("storedTheme", JSON.stringify(theme));
+    document.body.classList.add(theme);
```
Comment (dev-prabhat) [inline src/hooks/context/theme-context.js:13]: This is really cool, I like this approach to set theme maybe I can also use the same approach in my next project. 
Good work

**Review by dev-prabhat — APPROVED** [review]
You can merge this PR , Really like the Dark mode setup approach

**Pull request merged**